### DISCUSSION TITLE
Accessing events in a `GameModeBase` blueprint

### DISCUSSION THREAD

**Discussion by PeterMHammond** [nxrighthere/UnrealCLR#96] [Help] [answered]
Is there a way to access the GameModeBase Blueprint, so I can use this as the central point for setting configurations for and bringing events in from the UnrealCLR project?
Thank you,

![2021-01-19 21_43_02-OGDemo5 - Unreal Editor](https://user-images.githubusercontent.com/74989099/105132271-830e8980-5a9f-11eb-95ce-ff417be1fc2f.png)

**Comment by nxrighthere** [comment]
Hey. This API is not exposed yet, please, see the [roadmap](https://github.com/users/nxrighthere/projects/5?fullscreen=true).

**Reply by PeterMHammond** [reply]
My apologies - yes I see it there now.  I did look through the api and discussion threads first, but I'll make sure I check the roadmap first thing next time.  Thank you for all your diligent work on this.

**Reply by nxrighthere** [reply]
No worries, now I know that someone needs it, so I can expose it with higher priority than before.

**Comment by nxrighthere** [comment, marked as answer]
[`GameModeBase`](https://github.com/nxrighthere/UnrealCLR/blob/master/API/GameModeBase.md) is exposed in https://github.com/nxrighthere/UnrealCLR/commit/bd3d98f26bab4b2f775042462c0ee745c0cb36a4. You can retrieve it using [`World.GetGameMode()`](https://github.com/nxrighthere/UnrealCLR/blob/master/API/World-GetGameMode().md) method.
